pkg/flare/archive.go

```diff
@@ -33,6 +34,36 @@ import (
 // SearchPaths is just an alias for a map of strings
 type SearchPaths map[string]string
 
+// PermsInfos holds permissions info about the files shipped
+// in the flare.
+// It is bound to a given flare by the tempDir/hostname fields.
+// The filemode field is the file mode which will be used to create
+// the temporary permissions.log file on filesystem before being written in the zip.
+type PermsInfos struct {
+	tempDir  string
```

Comment:
I think I would remove `tempDir` from the structure and just pass the relevant info to the commit/write functions? Like, commit could take the path to the destination file or something along those lines.

Reply:
Ok! Not a problem and makes sense providing this info during the commit phase.